tests/test_roman_image_simulation.py

```diff
@@ -46,10 +47,19 @@
 }
 
 BAND = "F106"
-
+source = Source(
```

Comment:
can we make a sharp image test by adding the two separate images resulting in the same as generating a single call for both sources (modulo the lens light)